src/common/android/packageNameResolver.ts

```diff
@@ -54,10 +54,10 @@ export class PackageNameResolver {
     }
 
     /**
-     * Gets the default package name, based on the applicaiton name.
+     * Gets the default package name, based on the application name.
      */
     private getDefaultPackageName(applicationName: string): string {
-        return PackageNameResolver.DefaultPackagePrefix + applicationName;
+        return PackageNameResolver.DefaultPackagePrefix + applicationName.toLowerCase();
```

Comment:
Why is this lower cased in the first place? I thought that we intentionally removed some lower-casing because package names are actually case sensitive?